include/ck_tile/ops/fmha/kernel/fmha_fwd_kernel.hpp

```diff
@@ -1499,14 +1499,28 @@ struct FmhaFwdKernel
 
             AttentionVariant variant;
             const auto variant_params = [&] {
+                const float scale_s = [&] {
+                    if constexpr(QScaleEnum == BlockAttentionQuantScaleEnum::PERTENSOR)
+                    {
+                        float q_descale = *(reinterpret_cast<const float*>(kargs.q_descale_ptr));
+                        float k_descale = *(reinterpret_cast<const float*>(kargs.k_descale_ptr));
+
+                        return kargs.scale_s * q_descale * k_descale;
```

Comment:
Actually, the best method is to pass q_descale * k_descale to s_acc_ element_func. Will passing it to s_acc_ element_func affect performance?